silverback/_cli.py

```diff
@@ -138,30 +144,49 @@ def run(cli_ctx, account, runner_class, recorder_class, max_exceptions, bot):
 def build(generate, path):
     """Generate Dockerfiles and build bot images"""
     if generate:
-        if not (path := Path.cwd() / path).exists():
+        if (
+            not (path := Path.cwd() / path).exists()
+            and not (path := Path.cwd() / "bot").exists()
+            and not (path := Path.cwd() / "bot.py").exists()
+        ):
             raise FileNotFoundError(
-                f"The bots directory '{path}' does not exist. "
-                "You should have a `{path}/` folder in the root of your project."
+                f"The bots directory '{path}', 'bot/' and 'bot.py' does not exist in your path. "
+                f"You should have a '{path}/' or 'bot/' folder, or a 'bot.py' file in the root "
+                "of your project."
             )
-        files = {file for file in path.iterdir() if file.is_file()}
+        if path.is_file():
+            dockerfile_content = DOCKERFILE_CONTENT
+            docker_filename = f"Dockerfile.{path.parent.name}-bot"
+            dockerfile_content += f"COPY {path.name}/ /app/bot\n"
+            _build_helper(docker_filename, dockerfile_content)
+            return
```

Comment:
Should we return if `--generate` is called? What are the pros/cons of continuing to execute the build after generation?